docs/coding-guidelines/javascript.md

```diff
@@ -31,21 +31,21 @@ We specify an [EditorConfig](http://editorconfig.org/) configuration and encoura
 Object declarations can be made on a single line if they are short (remember the line length guidelines). When an object declaration is too long to fit on one line, there must be one property per line. Property names only need to be quoted if they are reserved words or contain special characters:
 
 ```js
-let map;
-
-// Preferred
-map = {
-	ready: 9,
-	when: 4,
-	'you are': 15
-};
-
-// Acceptable for small objects
-map = { ready: 9, when: 4, 'you are': 15 };
-
 // Bad
-map = { ready: 9,
-	when: 4, 'you are': 15 };
+const labels = { facebook: 'Facebook',
+	twitter: 'Twitter', 'google-plus': 'Google Plus' };
```

Comment:
haha, I almost made a comment like, "what are you thinking?" and then I noticed this is the "bad" example :wink: